domain-management/src/main/java/org/jboss/as/domain/management/security/PropertiesFileLoader.java

```diff
@@ -225,6 +225,41 @@ public synchronized void persistProperties() throws IOException {
         }
     }
 
+    protected String cleanKey(final String key) {
+        char[] keyChars = key.toCharArray();
+        char[] cleaned = new char[keyChars.length];
+        int keyPos = 0;
+        int cleanedPos = 0;
+
+        while (keyPos < keyChars.length) {
+            if (keyChars[keyPos++] == '\\') {
```

Comment:
Never mind.

I was going to ask a similar question about the default block at L250; whether it should fail if the next char after \ isn't \ or =.

But both here and at L250 looking at how java.util.Properties works it's basically the same as what you have here, so it's ok. I don't see a need to fail more nicely than the jdk does.